cli.py

```diff
@@ -19,9 +19,15 @@ def calculate(operation, integers):
     return calc.multiply(integers)
   if (operation == 'average'):
     return calc.average(integers)
+  if (operation == 'cm_to_m'):
+    return calc.cm_to_metres(integers)
```

Comment:
use Python3 joins, to convert a list to string and then return the string.

Reply:
Would this work?

Say `integers = [12, 14, 16]` and their corresponding meter values are  [0.12, 0.14, 0.16]

```
if (operation == 'cm_to_m'):
    m_results =  calc.cm_to_metres(integers)
    for i in range(len(m_results)):
         res_string += str(integers[i]) + "cm = " + str(m_results[i]) + "m\n"

    return(res_string)
```

So finally when the results are printed out, the user gets this:

```
------
12cm = 0.12m
14cm = 0.14m
16cm = 0.16m
------

```